sunpy/net/tests/test_fido.py

```diff
@@ -209,65 +205,36 @@ def test_unifiedresponse_slicing_reverse():
 def test_tables_single_response():
     results = Fido.search(
         a.Time("2012/1/1", "2012/1/5"), a.Instrument.lyra, a.Level.two)
-    tables = results.tables
+
+    with pytest.warns(SunpyDeprecationWarning):
+        tables = results.tables
 
     assert isinstance(tables, list)
     assert isinstance(tables[0], Table)
     assert len(tables) == 1
 
-    columns = ['Start Time', 'End Time', 'Instrument', 'Physobs',
-               'Source', 'Provider', 'Level']
-    assert columns == tables[0].colnames
```

Comment:
Why has this test gone?

Reply:
because we are no longer restricting the columns in the display table to this short list, we take any columns the client wants to give us.